source/PrintDiamond.cpp

```diff
@@ -0,0 +1,86 @@
+#include "PrintDiamond.h"
+
```

Comment:
Please change your VS configuration to use 4 spaces per indentation and no tabs. There have been religious wars about tabs vs. spaces, but 4 spaces is what we use here at Bluefruit. :)